cdserver/config/application.rb

```diff
@@ -27,6 +29,6 @@ class Application < Rails::Application
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
 
-    config.middleware.insert_after "Rails::Rack::Logger", "MiddlewareHealthcheck"
+    config.middleware.insert_after Rails::Rack::Logger, MiddlewareHealthcheck
```

Comment:
see above comment, you will likely need the quotes (read the article and see what works)
Looks like you can do this to see if it is auto-loading:
.... you can see the middleware stack by running
`rake middleware`
in your command line.